test/fixtures/circular-example.json

```diff
@@ -122,6 +162,66 @@
                                        \"id\": \"ORCL\",\n                                                                                          \"user\": {\n                                                                                            \"data\": {\n                                                                                              \"name\": \"doe\",\n                                                                                              \"company\": {\n                                                                                                \"data\": {}\n                                                                                              }\n                                                                                            }\n                                                                                          }\n                                                                                        }\n                                                                                      }\n                                                                                    }\n                                                                                  }\n                                                                                }\n                                                                              }\n                                                                            }\n                                                                          }\n                                                                        }\n                                                                      }\n                                                                    }\n                                                                  }\n                                                                }\n                                                              }\n                                                            }\n                                                          }\n                                                        }\n                                                      }\n                                                    }\n                                                  }\n                                                }\n                                              }\n                                            }\n                                          }\n                                        }\n                                      }\n                                    }\n                                  }\n                                }\n                              }\n                            }\n                          }\n                        }\n                      }\n                    }\n                  }\n                }\n              }\n            }\n          }\n        }\n      }\n    }\n  }\n}"
```

Comment:
Just a note, faker is rendering circular JSON to a certain depth so these will always be consistently the same.